sys/include/saul_reg.h

```diff
@@ -11,11 +11,13 @@
  * @ingroup     sys
  * @brief       Global sensor/actuator registry for SAUL devices
  *
+ * See also \ref drivers_saul
```

Comment:
You should be able to use @ instead of \ like for the other doxygen commands, to match the coding style of RIOT.